Patch to add Solaris 12 to the list of supported OSes.  This patch has
not yet been submitted upstream.

--- nanliu-staging-1.0.3/metadata.json.orig   2015-04-16 14:55:41.669164796 -0600
+++ nanliu-staging-1.0.3/metadata.json       2015-04-16 14:55:49.787256867 -0600
@@ -64,7 +64,8 @@
       "operatingsystem": "Solaris",
       "operatingsystemrelease": [
         "10",
-        "11"
+        "11",
+        "12"
       ]
     },
     {
